src/angular/11-declarative-state/restaurant.component.ts

```diff
@@ -1,85 +1,157 @@
-import { Component, OnInit } from '@angular/core';
-import { BehaviorSubject, Observable, of, combineLatest, merge } from 'rxjs';
-import { startWith, map, flatMap, shareReplay } from 'rxjs/operators';
-
-import { RestaurantService, ResponseData, State, City } from './restaurant.service';
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import {
+  combineLatest,
+  map,
+  mergeMap,
+  Observable,
+  of,
+  pairwise,
+  shareReplay,
+  startWith,
+  Subject,
+  takeUntil,
+  tap,
```

Comment:
are we using alll these operators in the code snippets? worried it's a lot to scroll trhough

Reply:
yes, and it is being auto formatted by prettier